x-pack/plugins/enterprise_search/public/applications/app_search/components/setup_guide/setup_guide.tsx

```diff
@@ -5,237 +5,58 @@
-                            'App Search and Kibana are on different authentication methods',
-                        }
-                      )}
-                      id="differentAuth"
-                      paddingSize="s"
-                    >
-                      <EuiText>
-                        <p>
-                          <FormattedMessage
-                            id="xpack.appSearch.troubleshooting.differentAuth.description"
-                            defaultMessage="This plugin does not currently support App Search and Kibana operating on different authentication methods (for example, App Search using a different SAML provider than Kibana)."
-                          />
-                        </p>
-                      </EuiText>
-                    </EuiAccordion>
-                    <EuiSpacer />
-                    <EuiAccordion
-                      buttonContent={i18n.translate(
-                        'xpack.appSearch.troubleshooting.standardAuth.title',
-                        {
-                          defaultMessage: 'App Search on Standard authentication',
-                        }
-                      )}
-                      id="standardAuth"
-                      paddingSize="s"
-                    >
-                      <EuiText>
-                        <p>
-                          <FormattedMessage
-                            id="xpack.appSearch.troubleshooting.standardAuth.description"
-                            defaultMessage='App Search operating on {standardAuthLink} is currently not fully supported by this plugin. Users created in App Search must be granted Kibana access. Users created in Kibana will see "Cannot find App Search account" error messages.'
-                            values={{
-                              standardAuthLink: (
-                                <EuiLink
-                                  href="https://swiftype.com/documentation/app-search/self-managed/security#standard"
-                                  target="_blank"
-                                >
-                                  Standard Auth
-                                </EuiLink>
-                              ),
-                            }}
-                          />
-                        </p>
-                      </EuiText>
-                    </EuiAccordion>
-                  </>
-                ),
-              },
-            ]}
-          />
-        </EuiPageContent>
-      </EuiPageBody>
-    </EuiPage>
-  );
-};
+export const SetupGuide: React.FC = () => (
+  <SetupGuideLayout
+    productName={i18n.translate('xpack.appSearch.productName', { defaultMessage: 'App Search' })}
+    productEuiIcon="logoAppSearch"
+    standardAuthLink="https://swiftype.com/documentation/app-search/self-managed/security#standard"
+    elasticsearchNativeAuthLink="https://swiftype.com/documentation/app-search/self-managed/security#elasticsearch-native-realm"
```

Comment:
I'm hoping we can potentially lose these two per-product props if I get approval from @chriscressman for opening up a PR for pulling out

https://www.elastic.co/guide/en/app-search/current/security-and-users.html
and
https://www.elastic.co/guide/en/workplace-search/current/workplace-search-security.html

into its own section in the general https://www.elastic.co/guide/en/enterprise-search/current/index.html docs 🤞

Reply:
@constancecchen - I'm not sure if you'd still like to make this change, but if so, I welcome that PR. I'm tracking a larger, related docs project in https://github.com/elastic/enterprise-search-pubs/issues/630, but I don't expect to act on it for a while. More broadly, I favor moving shared concerns out of product-specific books and into the Enterprise Search book.

If you proceed, here are some considerations:

- @jonas updated the Workplace Search version of the doc for 7.7.0, so it's more likely to be up to date than the App Search version.
- The existing Workplace Search URL will need to be redirected. In the PR, you can do a manual redirect (have the old doc link to the new), and then after it's deployed, I can get the website team to implement an HTTP redirect.
- The existing Workplace Search page is part of an "Administration & management" section, so you may need to modify the contents of the other docs in that section if they depend on the doc that's moving (I haven't checked).
- You'll have to find any links to the product-specific docs in the App Search and Workplace Search books and update them to point to the Enterprise Search book.

I wish moving docs wasn't so hard :cry: